src/WinRT.Runtime/ApiCompatBaseline.net5.0.txt

```diff
@@ -1,4 +1,8 @@
 Compat issues with assembly WinRT.Runtime:
+CannotRemoveAttribute : Attribute 'System.Runtime.CompilerServices.CompilerGeneratedAttribute' exists on 'WinRT.IInspectable.WinRT.IWinRTObject.get_AdditionalTypeData()' in the contract but not the implementation.

```

Comment:
This is just an attribute change so I do not think this is breaking or requires an assembly version change.